### PR TITLE
Added new AprilTagPipeline to only see tags 4 and 7 (for SimpleVision autos) and separated 2, 3, and 4-Score autos into Blue and Red sides

### PR DESCRIPTION
**Pull request by Team20DriverStation**

### CODE REVIEW

**Review by JamisOrr — APPROVED** [review]
Tested, lets gooooo autos

File: src/main/java/frc/robot/ClampedController.java [hunk 1]
```diff
@@ -24,11 +24,20 @@ public ClampedController(double p, double minPower, double maxPower) {
 
 	@Override
 	public double calculate(double currentValue, double setpoint) {
-		return clamp(super.calculate(currentValue, setpoint));
+		double power = clamp(super.calculate(currentValue));
+		super.setSetpoint(setpoint);
+		if (super.atSetpoint()) {
+			power = 0;
+		}
+		return power;
```
Comment (Gold856) [inline src/main/java/frc/robot/ClampedController.java:32]: This feels off

**Review by Gold856 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/ClampedController.java (hunk 1 above)
Comment (JamisOrr) [reply src/main/java/frc/robot/ClampedController.java:32]: Its not

Comment (JamisOrr) [reply src/main/java/frc/robot/ClampedController.java:32]: This is important for drive PIDs that control multiple things at once, it negates the min speed when close, very important

**Review by Gold856 — APPROVED** [review]

**Pull request merged**